webf/example/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   webf: ^0.10.0
-  webf_websocket: ^1.0.0
+  webf_websocket: ^1.1.0-beta.2
```

Comment:
update webf_websocket repo?

Reply:
Yes, because of webf.invokeModule API had changed.